tests/testthat/test_cloneVariable.R

```diff
@@ -19,3 +19,7 @@ test_that("Clone variable", {
 
 })
 
+test_that("Append varLabel", {
+  out <- cloneVariable(dfSAV, varName = "VAR1", new_varName = "VAR1_new", label_suffix = "(recoded)")
+  expect_equal(out$labels[8, 2], c("Variable 1 (recoded)"))
```

Comment:
Nit: Again, for the sake of completeness: Maybe add tests for the copies of the variable, too?

However, the previous test "Clone variable" does also check for successful cloning (with multiple codes / labels). Therefore, this might be redundant here.

Version 1: 

```r
expect_equal(out$labels[9, 2], c("Variable 1 (recoded)"))
expect_equal(out$labels[10, 2], c("Variable 1 (recoded)"))
```

Version 2:

```r
expect_equal(out$labels[c(8:10), 2], rep("Variable 1 (recoded)", 3))
```

Reply:
added version 2. Did not add further tests for the variables itself as this is tested in the tests before.